app/src/org/commcare/activities/NoCommCareActivity.java

```diff
@@ -0,0 +1,23 @@
+package org.commcare.activities;
+
+import android.os.Build;
+import android.os.Bundle;
+
+import org.commcare.utils.AndroidUtil;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.view.WindowInsetsControllerCompat;
+
+public class NoCommCareActivity extends AppCompatActivity {
```

Comment:
Think we should have `CommCareActivity` also extend from this so that we only put this logic in one place, also might make sense to name it `CommonBaseActivity` if we do that.